src/lib/DynamoDB.js

```diff
@@ -0,0 +1,137 @@
+import { 
+  BatchWriteItemCommand,
+  CreateTableCommand, 
+  DeleteTableCommand, 
+  DescribeTableCommand, 
+  DynamoDBClient, 
+  GetItemCommand, 
+  ListTablesCommand, 
+  QueryCommand, 
+  ScanCommand, 
+  UpdateTableCommand,
+} from "@aws-sdk/client-dynamodb";
+import { 
+  DeleteCommand, 
+  DynamoDBDocumentClient, 
+  PutCommand, 
+  UpdateCommand, 
+} from '@aws-sdk/lib-dynamodb';
+  
+const DYNAMO_DB_AWS_ACCESS_KEY =
+    process.env.KS_DYNAMO_DB_AWS_ACCESS_KEY_ID ??
+    process.env.KS_AWS_ACCESS_KEY_ID ??
+    '';
+const DYNAMO_DB_AWS_SECRET_ACCESS_KEY =
+    process.env.KS_DYNAMO_DB_AWS_SECRET_ACCESS_KEY ??
+    process.env.KS_AWS_SECRET_ACCESS_KEY ??
+    '';
+const DYNAMO_DB_AWS_REGION =
+    process.env.KS_DYNAMO_DB_AWS_REGION ??
+    process.env.KS_AWS_REGION ??
+    'us-east-1';
+  
+const marshallOptions = {
+    convertEmptyValues: false, // Whether to automatically convert empty strings, blobs, and sets to `null`. false, by default. 
+    removeUndefinedValues: true, // Whether to remove undefined values while marshalling. false, by default.
+    convertClassInstanceToMap: false, // Whether to convert typeof object to map attribute. false, by default.
+};
+      
+const unmarshallOptions = {
+    wrapNumbers: false, // Whether to return numbers as a string instead of converting them to native JavaScript numbers. false, by default.
+};
+  
+class DB {
```

Comment:
As per convention, the class name should be the filename as well.
```suggestion
class DynamoDB {
```